src/console_cmds.cpp

```diff
@@ -1422,8 +1414,8 @@ DEF_CONSOLE_CMD(ConAlias)
 DEF_CONSOLE_CMD(ConScreenShot)
 {
 	if (argc == 0) {
-		IConsoleHelp("Create a screenshot of the game. Usage: 'screenshot [viewport | normal | big | giant | heightmap | minimap] [no_con] [size <width> <height>] [<filename>]'");
-		IConsoleHelp("'viewport' (default) makes a screenshot of the current viewport (including menus, windows, ..), "
+		IConsolePrint(CC_HELP, "Create a screenshot of the game. Usage: 'screenshot [viewport | normal | big | giant | heightmap | minimap] [no_con] [size <width> <height>] [<filename>]'");
+		IConsolePrint(CC_HELP, "'viewport' (default) makes a screenshot of the current viewport (including menus, windows, ..), "
```

Comment:
guess this should in fact be multiple print statements now? It was to avoid the silly `-` in front of it :P